### PR TITLE
newsletter_send_processing.php: swap From and Reply-To

### PR DESCRIPTION
**Pull request by hemberger**
When sending newsletters in bulk, we want the From address to be
consistent (newsletter@smrealms.de) and the Reply-To address to
be unique (bounce_<account_id>@smrealms.de) for the purpose of
unsubscribing from e-mails.

This is recommended by Google's bulk sender guidelines.

### CODE REVIEW

File: admin/Default/newsletter_send_processing.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 //$mail->ConfirmReadingTo       = 'newsletter-read@smrealms.de';
 
-$mail->AddReplyTo('newsletter@smrealms.de', 'SMR Support');
+$mail->AddReplyTo('support@smrealms.de', 'SMR Support');
```
Comment (Page-) [inline admin/Default/newsletter_send_processing.php]: This line should be unnecessary right? Because it's always overridden below?

Comment (hemberger) [reply admin/Default/newsletter_send_processing.php]: When you send a newsletter to a single recipient, it does not get overwritten.

Comment (Page-) [reply admin/Default/newsletter_send_processing.php]: Ah ok, it might be worth moving it into that branch then to make the distinction clear

Comment (hemberger) [reply admin/Default/newsletter_send_processing.php]: Good idea. I've made the change in PR #340

**Pull request merged**